style.css

```diff
@@ -370,10 +400,178 @@ body {
   border: 1px solid  #cfd8dc;
 }
 
+textarea {
+  resize: none;
+}
+
 .home-indicator {
   background-color: #0d151e;
   height: 4px;
   width: 134px;
   margin: 0 0 10px 0;
   border-radius: 10px;
 }
+
+@media screen and (min-width: 1024px) {
+  .header {
+    padding-left: 100px;
+    padding-right: 100px;
+  }
+
+  .hamburger-menu {
+    display: none;
+  }
+
+  .nav-list {
+    list-style: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  .nav-link-item {
+    margin-right: 25px;
+  }
+
+  .nav-link {
+    text-decoration: none;
+    color: #344563;
+    font-size: 15px;
+  }
+
+  .profile-section {
+    background-image: url("assets/header-bg-desktop.png");
+    background-size: contain;
+    padding: 0 250px 0 250px;
+    min-height: 750px;
+    box-sizing: border-box;
+  }
+
+  .about-header {
+    font-size: 48px;
+    line-height: 60px;
+    width: 400px;
+    box-sizing: border-box;
+    color: #091e42;
+    padding-top: 200px;
+  }
+
+  .about-body {
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    width: 100%;
+  }
+
+  .post-section {
+    padding-left: 100px;
+    padding-right: 100px;
+  }
+
+  .sm {
+    display: none;
+    visibility: hidden;
+  }
+
+  .ms {
+    display: inline-block;
+    visibility: visible;
+  }
+
+  .post-container {
+    display: flex;
+    justify-content: space-between;
+    box-sizing: border-box;
+    margin: 0 auto;
+  }
+
+  .post-container > div {
+    margin: 0 0 0 0;
+  }
+
+  .post-container div:last-child {
+    padding-left: 20px;
+    padding-right: 20px;
+  }
+
+  .post-container .left {
+    flex: 1 1 50%;
+    padding: 0;
+  }
+
+  .image {
+    object-fit: contain;
+    height: 100%;
+  }
+
+  .post-container .right {
+    width: 50%;
+    padding: 10px 0 0 0;
+  }
+
+  .order-1 {
+    order: 1;
+  }
+
+  .order-2 {
+    order: 2;
+  }
+
+  .about-section {
+    padding-left: 100px;
+    padding-right: 100px;
+  }
+
+  .about-myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-column-gap: 20px;
+    min-height: auto;
+  }
```

Comment:
- In the `About Myself` section, you've paid adequate attention to the colors, spacings, and typography of this section :+1: 
However, according to the project requirement, the left block of this section takes 40% and the right covers the remaining 60% of the page's width. In your case, you're having the same width (1fr, 1fr) as the 2 blocks/columns of this section. Kindly fix it :blush: 

- Also, looking at the topics of each group in the `Skillset` section (talking about `Languages`, `Frameworks`, and `Skills`), your version is having more weight than what we have in the design. Check the images below to see the difference and please fix it as appropriate.

| Figma Design | Your Portfolio |
| ------------------ | ------------------ |
| ![original](https://user-images.githubusercontent.com/26856491/121499303-2cb1d100-c9d5-11eb-9393-98c07eccbdfe.png) | ![clone](https://user-images.githubusercontent.com/26856491/121499332-320f1b80-c9d5-11eb-9f68-fbe0152df27a.png) |